### PR TITLE
Add zero size objects support

### LINKED ISSUE #74: Add zero size objects support in test scenarios
Current test scenarios don't support load with empty payload, i.e., objects with zero size. 

That option would be useful to test load with 0 size objects as a boundary case. 

Need to add support both for GRPC and S3 protocols.

### PR DESCRIPTION
**Pull request by tatiana-nspcc**
Close #74.

### CODE REVIEW

File: scenarios/preset/helpers/neofs_cli.py [hunk 1]
```diff
@@ -13,18 +13,17 @@ def create_container(endpoint, policy, wallet_file, wallet_config):
...
-            return False
-        splitted = fst_str.split(": ")
-        if len(splitted) != 2:
-            raise ValueError(f"no CID was parsed from command output: \t{fst_str}")
-
-        print(f"Created container: {splitted[1]}")
-
-        return splitted[1]
+        # Regular expression to find the container ID, case-insensitive
+        pattern = r"container ID: ([A-Za-z0-9]+)"
```
Comment (roman-khimov) [inline scenarios/preset/helpers/neofs_cli.py:17]: There is a certain length that can be used to strengthen this regex (or it can collide with something else, `dog` matches it as well). It can be `([A-Za-z0-9]{43,44})` (in fact, 0 and some other symbols are incorrect as well, but that doesn't matter much).

File: scenarios/preset/helpers/neofs_cli.py [hunk 2]
```diff
@@ -13,18 +13,17 @@ def create_container(endpoint, policy, wallet_file, wallet_config):
...
+
+        # Search for the pattern in the output text with case-insensitive flag
+        match = re.search(pattern, output, re.IGNORECASE)
```
Comment (roman-khimov) [inline scenarios/preset/helpers/neofs_cli.py:20]: Why `IGNORECASE`? You have `A-Za-z` anyway.

Comment (tatiana-nspcc) [reply scenarios/preset/helpers/neofs_cli.py:20]: That's because "container ID" can be written in any case. The `re.IGNORECASE` flag ensures that the search is case-insensitive, allowing "container ID:" to match any case variation in the input text.

File: scenarios/preset/helpers/neofs_cli.py (hunk 1 above)
Comment (tatiana-nspcc) [reply scenarios/preset/helpers/neofs_cli.py]: Added length.

**Review by roman-khimov — APPROVED** [review]
